tests/test/main.cpp

```diff
@@ -4,6 +4,21 @@
 #include <catch2/catch.hpp>
 #include <faabric/util/logging.h>
 
+struct LogListener : Catch::TestEventListenerBase
+{
+    using TestEventListenerBase::TestEventListenerBase;
+
+    void testCaseStarting(Catch::TestCaseInfo const& testInfo) override
+    {
+        auto logger = faabric::util::getLogger();
+        logger->debug("---------------------------------------------");
+        logger->debug("TEST: {}", testInfo.name);
+        logger->debug("---------------------------------------------");
+    }
+};
```

Comment:
As in Faabric, I've added this to print test names. Without it its really difficult to parse the stream of test logging.